Fix to allow to run on 3d images

emiglietta opened the pull request:
- Adds a new parameter to be passed to the Cellpose call when running on volumetric images: `z_axiz'
- Allows volumetric display 
- Avoids showind outlines when doing volumetric display to escape shape mismatch error (third panel still shows the raw image but without outlines)

emiglietta committed:
add z_axis parameter to allow to run on 3D images

this setting errors if display is enabled bc display does not currently support 3d, but the module does run

emiglietta committed:
make the display 3D

this works but outline display still fails bc os shape mismatch

emiglietta committed:
avoid displaying labels when processing 3d images

otherwise there is a shape mismatch error.
The third panel still shows the original image, but without the outlines

bethac07 reviewed (approved):
Please test if you haven't already that this doesn't break 2D Cellpose4 support (I have no reason to think it does, just out of caution), feel free to do once pulled. Nice work!

emiglietta committed:
implement Erin's change to work with custom models

emiglietta committed:
fix indentation in remove obj from edges setting

emiglietta committed:
prob image fix

correct prob image size to x (original image) instead of y (found objects), in case no objects are found.
Also, set mask explicitly for probabilities.

emiglietta committed:
Merge branch 'master' into runcellpose_3d
